lib/executor/ramping_arrival_rate.go

```diff
@@ -316,25 +316,32 @@ func (varr RampingArrivalRate) Run(parentCtx context.Context, out chan<- stats.S
 	returnedVUs := make(chan struct{})
 	startTime, maxDurationCtx, regDurationCtx, cancel := getDurationContexts(parentCtx, duration, gracefulStop)
 
+	vusPool := newActiveVUPool()
+
 	defer func() {
+		vusPool.Close()
 		// Make sure all VUs aren't executing iterations anymore, for the cancel()
 		// below to deactivate them.
 		<-returnedVUs
 		cancel()
 		activeVUsWg.Wait()
 	}()
```

Comment:
This should prevent the race from `https://github.com/k6io/k6/pull/1957/checks?check_run_id=2631773502`
```
==================
WARNING: DATA RACE
Read at 0x00c000062078 by goroutine 128:
  internal/race.Read()
      /opt/hostedtoolcache/go/1.16.4/x64/src/internal/race/race.go:37 +0x206
  sync.(*WaitGroup).Add()
      /opt/hostedtoolcache/go/1.16.4/x64/src/sync/waitgroup.go:71 +0x219
  go.k6.io/k6/lib/executor.(*activeVUPool).AddVU()
      /home/runner/work/k6/k6/lib/executor/ramping_arrival_rate.go:519 +0x64
  go.k6.io/k6/lib/executor.RampingArrivalRate.Run.func3()
      /home/runner/work/k6/k6/lib/executor/ramping_arrival_rate.go:361 +0x290
  go.k6.io/k6/lib/executor.RampingArrivalRate.Run.func4()
      /home/runner/work/k6/k6/lib/executor/ramping_arrival_rate.go:379 +0x571

Previous write at 0x00c000062078 by goroutine 34:
  internal/race.Write()
      /opt/hostedtoolcache/go/1.16.4/x64/src/internal/race/race.go:41 +0x125
  sync.(*WaitGroup).Wait()
      /opt/hostedtoolcache/go/1.16.4/x64/src/sync/waitgroup.go:128 +0x126
  go.k6.io/k6/lib/executor.(*activeVUPool).Close()
      /home/runner/work/k6/k6/lib/executor/ramping_arrival_rate.go:535 +0x77
  go.k6.io/k6/lib/executor.RampingArrivalRate.Run.func1()
      /home/runner/work/k6/k6/lib/executor/ramping_arrival_rate.go:339 +0x54
  go.k6.io/k6/lib/executor.RampingArrivalRate.Run()
      /home/runner/work/k6/k6/lib/executor/ramping_arrival_rate.go:481 +0x20d0
  go.k6.io/k6/lib/executor.(*RampingArrivalRate).Run()
      <autogenerated>:1 +0x127
  go.k6.io/k6/lib/executor.TestRampingArrivalRateRunUnplannedVUs()
      /home/runner/work/k6/k6/lib/executor/ramping_arrival_rate_test.go:189 +0x7ee
  testing.tRunner()
      /opt/hostedtoolcache/go/1.16.4/x64/src/testing/testing.go:1193 +0x202
```

As far as I can see the waitgroup was empty at the time as a VU wasn't executing but it was *just* starting to and it raced between the `Add` and the `Close`. This change makes certain we no longer *will* try to "Add" before we close 
```suggestion
		// Make sure all VUs aren't executing iterations anymore, for the cancel()
		// below to deactivate them.
		<-returnedVUs
		cancel()
		vusPool.Close()
		activeVUsWg.Wait()
	}()
```